src/cli/commands/public-cmds/show-cmd.js

```diff
@@ -16,6 +16,7 @@ export default class Show extends Command {
   opts = [
     ['j', 'json', 'return a json version of the component'],
     ['r', 'remote', 'show a remote component'],
+    ['k', 'specifickeys <specifickeys>', 'get specific keys from json'],
```

Comment:
- [ ] change the name to fields 
- [ ] get specific keys from json -> if --json specified, returns specific fields (comma separated list)